### ISSUE TITLE
SD card not writable on FMUv1

### ISSUE THREAD

**Issue by LorenzMeier** [PX4/Firmware#1870]
Steps to reproduce:

  * Flash master, ensure SYS_AUTOSTART is set to zero
  * Run: ```mkdir /fs/microsd/testdir```

Top:
```
Processes: 13 total, 2 running, 11 sleeping
CPU usage: 7.26% tasks, 0.39% sched, 92.35% idle
Uptime: 223.217s total, 209.185s idle

 PID COMMAND                   CPU(ms) CPU(%)  USED/STACK PRIO(BASE) STATE 
   0 Idle Task                  209184 92.345     0/    0   0 (  0)  READY 
   1 hpwork                       4822  2.158   828/ 1792 192 (192)  w:sig 
   2 lpwork                        575  0.196   364/ 1792  50 ( 50)  w:sig 
   3 init                          167  0.000  1036/ 2592 100 (100)  w:sem 
   6 nshterm                        17  0.000   996/ 1496  70 ( 70)  w:sem 
  75 top                            52  0.588  1220/ 1696 100 (100)  RUN   
  28 commander                     330  0.098  2708/ 3192 215 (215)  w:sig 
  29 commander_low_prio             30  0.000   532/ 2592  50 ( 50)  w:sem 
  31 mavlink_if0                  1338  0.588  1764/ 2696 100 (100)  READY 
  36 mavlink_rcv_if0                14  0.000   780/ 2096 175 (175)  w:sem 
  53 sensors_task                 4348  1.962  1148/ 1992 250 (250)  w:sem 
  59 gps                           993  1.177   692/ 1496 220 (220)  w:sem 
  69 navigator                    1105  0.490   732/ 1792 120 (120)  w:sem 
```

free:

```
nsh> free
             total       used       free    largest
Mem:        173696      92128      81568      68656
```

**Comment by LorenzMeier** [comment]
Listing still works:
```
nsh> ls /fs/microsd
/fs/microsd:
 bootlog.txt
 dataman
 msgs_1971_03_21_02_42_11.txt
```
Read returns no content:

```
nsh> cat /fs/microsd/msgs_1971_03_21_02_42_11.txt
nsh> cat /fs/microsd/bootlog.txt
```

**Comment by LorenzMeier** [comment]
@kd0aij Taking care of this now.

@sjwilks I'm confused why this all worked for you.

**Comment by LorenzMeier** [comment]
Adding FMUv1 config to Dronetest now, so we will see these things coming in the future.

**Comment by LorenzMeier** [comment]
Another datapoint: df doesn't show the disk:
```
nsh> df
  Block  Number
  Size   Blocks     Used Available Mounted on
   128      823      823         0 /etc
```

**Comment by LorenzMeier** [comment]
This is all false alarm. I messed up the NuttX submodule and was missing a fix to SPI, which was in earlier. Verified with two boards. Closing.

**Comment by LorenzMeier** [comment]
Fixed via 6ad5243a516af1b3d12c826d4bf4dc30c6f6b11d.
